taichi/gui/cocoa.cpp

```diff
@@ -372,6 +373,14 @@ void GUI::process_event() {
           }
           break;
         }
+        case NSEventTypeScrollWheel: {
+          set_mouse_pos(p.x, p.y);
+          const auto dx = (int)cast_call<CGFloat>(event, "scrollingDeltaX");
+          const auto dy = (int)cast_call<CGFloat>(event, "scrollingDeltaY");
```

Comment:
Is the scale of scrollingDelta around 120?

Reply:
It depends on how hard you scroll, so I don't think we need to dictate a value (at my normal scroll strength it was around 50~70?). In addition, Mac has this ease-out animation effect so that the delta gradually decays to zero...